lib/immutable-struct.rb

```diff
@@ -64,12 +64,22 @@ def self.new(*attributes,&block)
           end
         end
       end
+
+      define_method(:==) do |other|
+        return unless other.is_a?(klass)
```

Comment:
Shouldn't this return a bool?  return with no value will return a nil.  Your README updates say you'll get false, but that is not quite right